src/main/java/com/pgs/booking/model/dto/CreateUpdateFlightDto.java

```diff
@@ -0,0 +1,25 @@
+package com.pgs.booking.model.dto;
+
+import lombok.Builder;
+import lombok.Value;
+
+import javax.validation.constraints.NotBlank;
+import java.time.LocalDateTime;
+
+@Value
```

Comment:
just a head-up: this might leat to similar problems as it did with passengers

Reply:
Sure, to add @Data and @NoArgsConstructor
@AllArgsConstructor instead of @Valid.